web/src/reducers/activities.test.js

```diff
@@ -19,12 +19,12 @@ describe('activities reducer', () => {
     name: '',
     start: '',
     files: [],
-    years: { '2018': 0, '2019': 0 },
+    years: { '2019': 0, '2020': 0 },
```

Comment:
Should we be setting these to the current year/year +1 dynamically?

Reply:
That'd be more robust, but I'm hesitant because it makes the tests nondeterministic.  I think in this case it's simple enough it'd probably be fine, but ideally I'd probably want to use Sinon's time mocking and force everything into a known year, regardless of the actual current year.  Then the tests would pass forever, and they'd be deterministic too.

I'm 100% on board if you think we should make this test more robust before we merge it, though.